src/pages/my_page/user_profile/MyProfile.tsx

```diff
@@ -74,140 +89,273 @@ export const MyProfile = () => {
+                          alignItems: 'center',
+                        }}>
+                        <Tooltip title={t('common.edit')}>
+                          <IconButton
+                            data-testid={dataTestId.myPage.myProfile.editPreferredNameButton}
+                            onClick={() => setEditPreferredNames(!editPreferredNames)}>
+                            <EditIcon sx={{ width: '1.2rem' }} />
+                          </IconButton>
+                        </Tooltip>
+                      </Box>
+                    </Grid>
+                    <Grid item xs={16} md={3}>
+                      <Typography fontWeight="bold" sx={typographyStyling}>
+                        {t('my_page.my_profile.identity.identity_numbers')}
+                      </Typography>
+                    </Grid>
+                    <Grid item xs={14} md={12}>
+                      <Box
+                        sx={{
+                          height: '100%',
+                          display: 'grid',
+                          gridTemplateColumns: '1fr 1fr',
+                          gap: '1rem',
+                          justifyContent: 'space-evenly',
+                          alignItems: 'center',
+                        }}>
+                        <NationalIdNumberField nationalId={user.nationalIdNumber} />
+                        <TextField
+                          value={getIdentifierFromId(user.cristinId ?? '')}
+                          disabled
+                          label={t('common.id')}
+                          size="small"
+                          variant="filled"
+                        />
+                      </Box>
+                    </Grid>
+                    <Grid item md={13}>
+                      <Box
+                        sx={{
+                          display: 'flex',
+                          flexDirection: 'column',
+                          alignItems: 'start',
+                          my: '0.5rem',
+                        }}>
+                        <Typography fontWeight={600}>{t('common.orcid')}</Typography>
+                        <UserOrcid user={user} />
+                      </Box>
+                    </Grid>
+                    <Grid item xs={16}>
+                      <Typography fontWeight="bold" fontSize={15}>
+                        {t('my_page.my_profile.contact_information')}
+                      </Typography>
+                    </Grid>
+                    <Grid item xs={16} md={3}>
+                      <Typography fontWeight="bold" sx={typographyStyling}>
+                        {t('my_page.my_profile.telephone')}
+                      </Typography>
+                    </Grid>
+                    <Grid item xs={14} md={12}>
+                      <Field name={'contactDetails.telephone'}>
+                        {({ field }: FieldProps<string>) => (
+                          <TextField
+                            {...field}
```

Comment:
bør nok ha noe validering på disse feltene. Enten via yup, men det burde holde med ren html også
```suggestion
                            {...field}
                            type="tel"
```

![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/e427e72e-7213-4890-acea-4c901522db0e)

Reply:
Jepp, skal legg inn skikkelig validering i egen PR 👍

Reply:
litt obs på at det kan gi problemer å dele opp sånt i egne PRs da. Om bruker får lagt inn en epost i dag, men får feilmelding når man åpner samme visning i morgen kan oppleves rart. Håper egentlig at APIet gir 400 error om man poster ugyldige verdier her også, men ikke sikkert det er tilfeller 🤔 Så man kan potensielt åpne for å lagre ugyldige verdier som bruker ikke nødvendigvis retter opp selv, og som lager problemer i fremtiden

Reply:
Men valideringen med `type` som er lagt til nå skal være nok for å hindre brukere i å lagre ugyldig data da, så holder det👍

Reply:
La inn yup-validering i denne pr'en istedet for å finne en løsning på andre ting som ville bli fikset av det uansett.